File: crossecore-java/src/java_/UnresolvedItemAccess.java
package java_;
public interface UnresolvedItemAccess 
 extends Expression,NamespaceAccess
{
	
	public static com.crossecore.ocl.QuickSet<UnresolvedItemAccess> allInstances_ = new com.crossecore.ocl.QuickSet<UnresolvedItemAccess>(UnresolvedItemAccess.class);
	
		
	public static com.crossecore.ocl.QuickSet<UnresolvedItemAccess> allInstances(){
		
		com.crossecore.ocl.QuickSet<UnresolvedItemAccess> result = new com.crossecore.ocl.QuickSet<UnresolvedItemAccess>(UnresolvedItemAccess.class);
		result.addAll(UnresolvedItemAccess.allInstances_);
		
		
		return result;
	}
	
	//public static com.crossecore.ocl.QuickSet<UnresolvedItemAccess> allInstances = new com.crossecore.ocl.QuickSet<UnresolvedItemAccess>(UnresolvedItemAccess.class);
	
	
	UnresolvedItem getElement();
	void setElement(UnresolvedItem value);
	
	ASTNode getQualifier();
	void setQualifier(ASTNode value);
	
}

File: crossecore-typescript/java_/UnresolvedItemAccess.ts
import {Set} from "ecore/Set";
import {Expression} from "./Expression";
import {UnresolvedItem} from "./UnresolvedItem";
import {ASTNode} from "./ASTNode";
import {NamespaceAccess} from "./NamespaceAccess";

export interface UnresolvedItemAccess
extends Expression,
NamespaceAccess

{
	
	element:UnresolvedItem;
	qualifier:ASTNode;
	

}

